src/pycram/robot_descriptions/turtlebot3_waffle_pi_description.py

```diff
@@ -0,0 +1,13 @@
+import rospkg
+
+from pycram.robot_description import RobotDescriptionManager, RobotDescription
+
+rospack = rospkg.RosPack()
+filename = rospack.get_path('pycram') + '/resources/robots/' + "turtlebot3_waffle_pi" + '.urdf'
```

Comment:
Can we maybe rename the robot to just turtle bot and have in the description that turtle bot == turtlebot3_waffle_pi cause i don't wanna write this all the time when using the robot